### DISCUSSION TITLE
Waters HDMSe Data

### DISCUSSION THREAD

**Discussion by charlietrypsin** [vdemichev/DiaNN#681] [Q&A]
Hi,

I've converted some waters HDMSe data into mzml, however it gets 0 hits from what looks like should be good data from the TIC. I know there was some discussion that SONAR data could work (https://github.com/vdemichev/DiaNN/discussions/101) but want to confirm that DIA-NN doesn't take HDMDSe data, and ask if there are any plans to add this feature in the future

Many thanks,

Charlie

**Comment by vdemichev** [comment]
Hi Charlie,

Native Waters format is not supported, there are no support plans so far.

Best,
Vadim

**Comment by RedOctoCat** [comment]
What does DIA-NN require from MZML files - either those created from Waters HDMSe data, or in general?

**Reply by vdemichev** [reply]
Need to contain index & peak-picked data & be in the 'normal DIA' format, i.e. without MSX-DIA etc
